jobs/config.json

```diff
@@ -5935,6 +5936,7 @@
       "--gke-environment=test",
       "--provider=gke",
       "--skew",
+      "--test_args=--ginkgo.skip=\\[Flaky\\]|\\[Feature:.+\\]|Initializers --minStartupPods=8",
```

Comment:
Do you need to specifically call out for "initializers"? The test description has the `[Feature:` label: https://github.com/kubernetes/kubernetes/blob/master/test/e2e/apimachinery/initializers.go#L40

Reply:
not from https://github.com/mbohlool/kubernetes/blob/b4742b51d9b7374fcdbe46d503547136ecadc773/test/e2e/extension/initializers.go#L37

Reply:
Maybe instead we should update that line?

Reply:
Let's go with whatever fix that's the easiest. This PR is lgtm.